dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -217,12 +206,23 @@ USER root
 # We don't use npx playwright install-deps because that could pull a newer version
 # Install additional fonts as well
 RUN for iter in {1..10}; do \
-    apt-get update -y && \
-    $NODE_PATH/node/lib/node_modules/@elastic/synthetics/node_modules/.bin/playwright install-deps chromium && \
-    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends --yes \
-    fonts-noto \
-    fonts-noto-cjk && \
-    exit_code=0 && break || exit_code=$? && echo "apt-get error: retry $iter in 10s" && sleep 10; \
+    microdnf -y update && \
+    microdnf -y install fontconfig freetype cairo glib2 gtk3 pango xorg-x11-fonts-misc xorg-x11-fonts-Type1 \
+    at-spi2-atk atk at-spi2-core alsa-lib cups-libs dbus-libs libdrm mesa-libEGL mesa-libgbm nspr nss libX11 \
+    libX11-xcb libxcb libXcomposite libXdamage libXext libXfixes libXrandr libxkbcommon libxshmfence glib2 \
+    dbus-glib libicu mesa-libGL unzip -y && \
+    mkdir -p /usr/share/fonts/google-noto && \
+    curl -LO https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJKjp-hinted.zip && \
+    unzip NotoSansCJKjp-hinted.zip -d /usr/share/fonts/google-noto && \
+    rm -f NotoSansCJKjp-hinted.zip && \
+    microdnf -y remove unzip && \
+    curl -LO https://github.com/googlefonts/noto-fonts/raw/main/hinted/ttf/NotoSans/NotoSans-Regular.ttf && \
+    mv NotoSans-Regular.ttf /usr/share/fonts/google-noto && \
+    curl -LO https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf && \
+    mv NotoColorEmoji.ttf /usr/share/fonts/google-noto && \
+    fc-cache -fv && \
+    microdnf clean all && \
+    exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; \
```

Comment:
This needs to be tested with running synthetics / monitors for Heartbeat.
The instructions can be found here https://www.elastic.co/guide/en/observability/8.15/uptime-get-started.html

Reply:
I'm not sure if it's normal. The testing instructions for the same functionality in Wolfi (https://github.com/elastic/beats/pull/40689) included browser monitors but had no mention of Kibana. I didn't perform this testing myself, so I don't know the exact steps.

@emilioalvap could you help us figure out what's missing? Perhaps something we do wrong?

Reply:
@pkoutsovasilis browser journeys use the synthetics-* datastreams, it's not enabled by default on the Uptime UI, you need to add it manually.

Reply:
thanks for the extra info @emilioalvap I did update the datastreams of upstream and now I can see also the Journey one (cc @rdner )

<img width="1571" alt="image" src="https://github.com/user-attachments/assets/22ab2958-ab19-44e3-a592-25c3edd6433d" />